### PR TITLE
[axof: 231] View section step

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Add a fifth step to our dialog
image

Clicking view section takes you to the assessments tab of the teacher dashboard in a new tab.

In the process, I generalized the way we get the code.org URL from studio-code.org JS.

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
@aoby might be interested in second commit here

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -237,6 +237,7 @@
   "reportAbuse": "Report Abuse",
   "resetProgram": "Reset",
   "restoreThisVersion": "Restore this Version",
+  "reviewResponses": "5. Go to the assessment/survey tab for your section to review student responses.",
```
Comment (islemaster) [inline apps/i18n/common/en_us.json]: Is it weird that the numbering is inline in the localization here? (as opposed to some kind of ordered list?)

**Comment by islemaster** [comment]
LGTM!

File: apps/i18n/common/en_us.json (hunk 1 above)
Comment (Bjvanminnen) [reply apps/i18n/common/en_us.json]: Good point. It might make more sense to extract the numbering (and have it not be localized).

**Pull request merged**